docs/designers-developers/developers/filters/block-filters.md

```diff
@@ -319,6 +319,51 @@ wp.hooks.addFilter( 'editor.BlockListBlock', 'my-plugin/with-client-id-class-nam
 ```
 {% end %}
 
+Adding new properties to the block's wrapper component can be achieved by adding them to the `wrapperProps` property of the returned component.
+
+_Example:_
+
+{% codetabs %}
+{% ESNext %}
+```js
+const { createHigherOrderComponent } = wp.compose;
+
+const withClientIdClassName = createHigherOrderComponent( ( BlockListBlock ) => {
+	return ( props ) => {
+		const wrapperProps = { 'data-your-property': 'your-value' };
+		return <BlockListBlock { ...props } wrapperProps={ wrapperProps } />;
```

Comment:
Thanks for contributing to the docs!

Seems like this would overwrite existing `wrapperProps`, could it be changed to something like:
```suggestion
		const wrapperProps = { 
		    ...props.wrapperProps, 
		    'data-your-property': 'your-value',
		};
		return <BlockListBlock { ...props } wrapperProps={ wrapperProps } />;
```

The ES5 example has the same issue